src/documentation/pages/components-wip/forms/radio-button/examples/_base.vue

```diff
@@ -26,30 +22,38 @@ import { Vue } from 'vue-facing-decorator';
     return {
       exampleTabs: [
         {
-          disabled: true,
+          active: true,
           id: 'webcomponent',
           label: 'Web Component',
         },
         {
-          active: true,
+          disabled: false,
           id: 'htmlblueprint',
           label: 'HTML Blueprint',
         },
       ],
       codeSnippets: {
-        webcomponent: ``,
+        webcomponent: `<fieldset>
+  <legend class="chi-label">Select an option</legend>
+  <div class="chi-form__item>
```

Comment:
There many occurrences of this:

```suggestion
  <div class="chi-form__item">
```